gnocchi/incoming/s3.py

```diff
@@ -27,11 +28,13 @@
 boto3 = s3.boto3
 botocore = s3.botocore
 
+LOG = daiquiri.getLogger(__name__)
+
 
 class S3Storage(incoming.IncomingDriver):
 
     # NOTE(gordc): override to follow s3 partitioning logic
-    SACK_NAME_FORMAT = "{number}-{total}/"
+    SACK_NAME_FORMAT = "{number}-{total}"
```

Comment:
should we have a releasenote that for sure s3 will lose unprocessed measures?

Reply:
Upgrade should always be done with unprocessed measures empty, that's in the docs.